mmv1/products/dataplex/Datascan.yaml

```diff
@@ -0,0 +1,640 @@
+                  name: 'rowConditionExpectation'
+                  description: |
+                    Table rule which evaluates whether each row passes the specified condition.
+                  properties:
+                    - !ruby/object:Api::Type::String
+                      name: 'sqlExpression'
+                      description: |
+                        The SQL expression.
+                - !ruby/object:Api::Type::NestedObject
+                  name: 'tableConditionExpectation'
+                  description: |
+                    Table rule which evaluates whether the provided expression is true.
+                  properties:
+                    - !ruby/object:Api::Type::String
+                      name: 'sqlExpression'
+                      description: |
+                        The SQL expression.
+            - !ruby/object:Api::Type::Boolean
+              name: 'passed'
+              description: |
+                Whether the rule passed or failed.
+            - !ruby/object:Api::Type::String
+              name: 'evaluatedCount'
+              default_values: 0
+              description: |
+                The number of rows a rule was evaluated against. This field is only valid for ColumnMap type rules.
+                Evaluated count can be configured to either
+                1. include all rows (default) - with null rows automatically failing rule evaluation, or
+                2. exclude null rows from the evaluatedCount, by setting ignore_nulls = true.
+            - !ruby/object:Api::Type::String
+              name: 'passedCount'
+              description: |
+                The number of rows which passed a rule evaluation. This field is only valid for ColumnMap type rules.
+            - !ruby/object:Api::Type::String
+              name: 'nullCount'
+              description: |
+                The number of rows with null values in the specified column.
+            - !ruby/object:Api::Type::Integer
+              name: 'passRatio'
+              description: |
+                The ratio of passedCount / evaluatedCount. This field is only valid for ColumnMap type rules.
+            - !ruby/object:Api::Type::String
+              name: 'failingRowsQuery'
+              description: |
+                The query to find rows that did not pass this rule. Only applies to ColumnMap and RowCondition rules.
+      - !ruby/object:Api::Type::String
+        name: 'rowCount'
+        description: |
+          The count of rows processed.
+      - !ruby/object:Api::Type::NestedObject
+        name: 'scannedData'
+        description: |
+          The data scanned for this result.
+        properties:
+          - !ruby/object:Api::Type::NestedObject
+            name: 'incrementalField'
+            description: |
+              The range denoted by values of an incremental field
+            properties:
+              - !ruby/object:Api::Type::string
```

Comment:
should we capitalize `string` here and with its siblings?

Reply:
+1